[2023-04-06T03:26:37.416+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: io_resto_casa.respond manual__2023-04-06T03:26:22.417437+00:00 [queued]>
[2023-04-06T03:26:37.430+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: io_resto_casa.respond manual__2023-04-06T03:26:22.417437+00:00 [queued]>
[2023-04-06T03:26:37.432+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T03:26:37.433+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-04-06T03:26:37.434+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-04-06T03:26:37.460+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): respond> on 2023-04-06 03:26:22.417437+00:00
[2023-04-06T03:26:37.467+0000] {standard_task_runner.py:55} INFO - Started process 700 to run task
[2023-04-06T03:26:37.472+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'io_resto_casa', 'respond', 'manual__2023-04-06T03:26:22.417437+00:00', '--job-id', '38', '--raw', '--subdir', 'DAGS_FOLDER/ciao.py', '--cfg-path', '/tmp/tmp_e7btq9u']
[2023-04-06T03:26:37.477+0000] {standard_task_runner.py:83} INFO - Job 38: Subtask respond
[2023-04-06T03:26:37.581+0000] {task_command.py:389} INFO - Running <TaskInstance: io_resto_casa.respond manual__2023-04-06T03:26:22.417437+00:00 [running]> on host cfece00ff772
[2023-04-06T03:26:37.695+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=io_resto_casa
AIRFLOW_CTX_TASK_ID=respond
AIRFLOW_CTX_EXECUTION_DATE=2023-04-06T03:26:22.417437+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-06T03:26:22.417437+00:00
[2023-04-06T03:26:37.700+0000] {python.py:177} INFO - Done. Returned value was: Greet Responded Again
[2023-04-06T03:26:37.731+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=io_resto_casa, task_id=respond, execution_date=20230406T032622, start_date=20230406T032637, end_date=20230406T032637
[2023-04-06T03:26:37.766+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-04-06T03:26:37.789+0000] {taskinstance.py:2596} INFO - 0 downstream tasks scheduled from follow-on schedule check
